Package.swift

```diff
@@ -0,0 +1,21 @@
+// swift-tools-version:5.0
+
+import PackageDescription
+
+let package = Package(
+    name: "RxSwiftExt",
+    platforms: [
+        .iOS(.v8), .tvOS(.v9), .macOS(.v10_11)
+    ],
+    products: [
+        .library(name: "RxSwiftExt", targets: ["RxSwiftExt"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", .upToNextMajor(from: "5.0.0")),
+    ],
+    targets: [
+        .target(name: "RxSwiftExt", dependencies: ["RxSwift", "RxCocoa"], path: "Source"),
+        .testTarget(name: "RxSwiftExtTests", dependencies: ["RxSwiftExt", "RxTest"], path: "Tests"),
+    ],
+    swiftLanguageVersions: [.v5]
```

Comment:
Don't remember this off the top of my head but should this indicate .v5_1 as well? I think we also support Swift 4.2 here.